bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/InitializeToolsStartup.java

```diff
@@ -134,7 +126,11 @@ else if (isInstallerConfigSet())
 			});
 		}
 
-		// read esp-idf.json file
+		updateEnvUsingIdfJsonFile(idf_json_file);
```

Comment:
even if we click No, we will get here after the second eclipse restart, am I missing something here?